### ISSUE TITLE
用coco val2017作为训练集，训练失败

### ISSUE THREAD

**Issue by pbdahzou** [kuku-sichuan/FPN_Faster_RCNN#2]
为快速测试，用coco val2017作为训练集，训练失败，输出信息如下。希望作者能帮忙看下是啥问题，多谢！

/usr/local/lib/python2.7/dist-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
WARNING:tensorflow:From train.py:154: replicate_model_fn (from tensorflow.contrib.estimator.python.estimator.replicate_model_fn) is deprecated and will be removed after 2018-05-31.
Instructions for updating:
Please use `tf.contrib.distribute.MirroredStrategy` instead.
INFO:tensorflow:Replicating the `model_fn` across ['/gpu:0'].  Variables are going to be placed on ['/gpu:0'].  Consolidation device is going to be /gpu:0.
INFO:tensorflow:Using config: {'_save_checkpoints_secs': None, '_global_id_in_cluster': 0, '_session_config': gpu_options {
  allow_growth: true
}
allow_soft_placement: true
, '_keep_checkpoint_max': 5, '_tf_random_seed': None, '_task_type': 'worker', '_train_distribute': None, '_is_chief': True, '_cluster_spec': <tensorflow.python.training.server_lib.ClusterSpec object at 0x7fd4cc17d050>, '_model_dir': './logs', '_num_worker_replicas': 1, '_task_id': 0, '_log_step_count_steps': 200, '_master': '', '_save_checkpoints_steps': 2500, '_keep_checkpoint_every_n_hours': 10000, '_evaluation_master': '', '_service': None, '_device_fn': None, '_save_summary_steps': 2500, '_num_ps_replicas': 0}
INFO:tensorflow:Calling model_fn.
WARNING:tensorflow:From train.py:102: get_or_create_global_step (from tensorflow.contrib.framework.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Please switch to tf.train.get_or_create_global_step
WARNING:tensorflow:From train.py:110: __init__ (from tensorflow.contrib.estimator.python.estimator.replicate_model_fn) is deprecated and will be removed after 2018-05-31.
Instructions for updating:
Please use `tf.contrib.distribute.MirroredStrategy` instead.
/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/gradients_impl.py:108: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from ./logs/model.ckpt-0
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Saving checkpoints for 0 into ./logs/model.ckpt.
INFO:tensorflow:Loss for final step: None.

**Comment by redzhepdx** [comment]
I faced with similar issue while I was trying to train my custom data. Could you explain that how did you resolve the issue ?
